packages/block-library/src/navigation/edit/index.js

```diff
@@ -690,6 +700,24 @@ function Navigation( {
 		</InspectorControls>
 	);
 
+	if ( isPlaceholder && CustomPlaceholder ) {
```

Comment:
We should explain _why_ the placeholder is still here even though generally it's deprecated.